test/roots/pillar/sudoers.sls

```diff
@@ -0,0 +1,51 @@
+sudoers:
+  users:
+    root:
+      - 'ALL=(root) NOPASSWD: ALL'
```

Comment:
I added root as a personal preference. Almost every OS defaults to giving root NOPASSWD:ALL - otherwise, someone will attempt to run sudo as root and be denied.